### ISSUE TITLE
verbose FOOOFGroup

### ISSUE THREAD

**Issue by srcole** [fooof-tools/fooof#67]
I'm running FOOOFGroup on 7500 power spectra, and it'd be nice to have some periodic output to get a sense of how many are done / how much time is remaining.

If I'm not mistaken, the current 'verbose' option only alerts the user of warnings, but I think it'd be useful to have some other options for more updates. e.g. if verbose = 2, then print an update after every 100 PSDs.

**Comment by TomDonoghue** [comment]
Yeh, I thought about adding a logger. It gets non-trivial to report when running in parallel though, so I didn't bother for now. 

Running such large FOOOFGroups is probably something of an edge case, so what I do is (if I'm running multiple FG's) is print reports per group, but not internally.

**Comment by srcole** [comment]
Understandable.

This seems to only a problem when I run FOOOF over a large frequency range (or dense frequency sampling), so it takes a long time, which probably isn't best.

Not an issue now that I've limited my frequency range more.

**Comment by TomDonoghue** [comment]
Work in progress (almost there, I think) on this now available on 'progbar' branch. 

Available to try out if anyone wants.

**Comment by TomDonoghue** [comment]
Merged in #112
